storage/aptosdb/src/lib.rs

```diff
@@ -484,19 +490,39 @@ impl AptosDB {
 
     pub fn open_dbs<P: AsRef<Path> + Clone>(
         db_root_path: P,
+        db_path_overrides: Option<DbPathConfig>,
         rocksdb_configs: RocksdbConfigs,
         readonly: bool,
         max_num_nodes_per_lru_cache_shard: usize,
     ) -> Result<(LedgerDb, StateMerkleDb, StateKvDb)> {
-        let ledger_db = LedgerDb::new(db_root_path.as_ref(), rocksdb_configs, readonly)?;
+        let ledger_db = LedgerDb::new(
+            db_path_overrides
```

Comment:
any chance that we provide some util functions to get these db pathes that other component can use?

Reply:
Rewrote, PTAL